py2hwsw/docker/README.md

```diff
@@ -33,13 +35,17 @@ make sim-run
 # NOTE: dependencies take some time to install the 1st time
 ```
 
+## Images in Github Container Registry
+- The `docker-image.yml` workflow builds and publishes the docker image on each
+  release
+- To pull and run the docker image:
+```bash
+# pull latest version
+docker run ghcr.io/iobundle/py2hwsw:latest
```

Comment:
This command is not working on my machine:
```Bash
$ docker run ghcr.io/iobundle/py2hwsw:latest
Unable to find image 'ghcr.io/iobundle/py2hwsw:latest' locally
docker: Error response from daemon: manifest unknown.
See 'docker run --help'.
```

I assume it's because we have not yet published docker images to this repo.

Reply:
Yes, that command will only work after a release is created.
There will be packages listed in https://github.com/IObundle/py2hwsw (right side column, below Releases)